src/routes/index.js

```diff
@@ -1 +1,2 @@
+import "./adminRoutes";
```

Comment:
then when you add your routes codes to fiyin's file, and you change the file name to `countryRoutes`, import it here/replace `adminRoutes`